src/tespy/components/heat_exchangers.py

```diff
@@ -2885,6 +2944,21 @@ def ttd_u_func(self):
         T_o2 = T_mix_ph(o2, T0=self.outl[1].T.val_SI)
         return self.ttd_u.val - T_bp_p(i1) + T_o2
 
+    def exergy_balance(self):
+        r"""
+        Perform exergy balance of a condenser.
+
+            .. math::
+
+                Ex_{output} = 0
+                Ex_{input} = \dot{m}_{in} \cdot (ex_{ph,in} - ex_{ph,out})
+                Ex_{loss} = 0
+        """
+        self.Ex_output = 0
```

Comment:
I do not really like `Ex_output = 0` here, as the condenser might be used for heat production in district heating. In that case, Ex is clearly not 0.